Global.h

```diff
@@ -698,3 +698,16 @@ namespace vstd
 }
 using vstd::operator-=;
 using vstd::make_unique;
+
+#if defined VCMI_ANDROID && defined NO_STD_TOSTRING

```

Comment:
OK. Let this this hack-fix be. But if we get more problems with STL on Android NDK, I will insists on proper solution.

P.S. some more explanation:

> Root Cause: The root cause seems to be that the C99 functionality usage has been disabled in the GNU stdlibc++ on the armeabi-v7a platform due to the fact the the Bionic libc does not support complex math (the standard C library on Android is Bionic).

http://stackoverflow.com/questions/17950814/how-to-use-stdstoul-and-stdstoull-in-android#18124627